### PR TITLE
add routing and detailed page

### PR DESCRIPTION
**Pull request by DanCesca**
Added routing and detailed page with "Show more"-button and logic

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+import { Link } from "react-router-dom";
 
 const CardWrapper = styled.section`
   display: flex;
   flex-direction: column;
   justify-content: space-evenly;
+  align-items: center;
 
   li {
     list-style: none;
```
Comment (mbosselmann) [inline src/components/Card.jsx:12]: If you add `list-style: none` to your `li` element then you should add as attribute `role="list"` to your `ul` element in line 38. This is for accessibility reasons as a screenreader can't identify a list when `list-style` ist set to `none`. But if you add the role attribute to your list it is fine. 👍

File: src/components/DetailedCard.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const CardWrapper = styled.section`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+
+  li {
+    list-style: none;
```
Comment (mbosselmann) [inline src/components/DetailedCard.jsx:11]: Same here for your `ul` element. :)

File: src/pages/CharacterPage.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import "../App.css";
+import Header from "../components/Header";
+import DetailedCard from "../components/DetailedCard";
+import NavBar from "../components/NavBar";
+
+export default function CharacterPage() {
+  
+  return (
+    <div className="App">
```
Comment (mbosselmann) [inline src/pages/CharacterPage.js:9]: You can also work here with a `styled component`instead of an extra `App.css`. :-)

**Review by mbosselmann — COMMENTED** [review]
Good job! Keep it up! 👍 I have two remarks you should consider before merging. :)

File: src/components/Card.jsx (hunk 1 above)
Comment (DanCesca) [reply src/components/Card.jsx:12]: Hey! Thank you very much for your comment. I will definitely add those attributes right after the merge directly in the main branch. I hope that's fine? :)

File: src/pages/CharacterPage.js (hunk 3 above)
Comment (DanCesca) [reply src/pages/CharacterPage.js:9]: Could I please keep it like that for now? :) Thanks!

**Pull request merged**